website/content/docs/platform/k8s/vso/installation.mdx

```diff
@@ -105,3 +105,57 @@ customresourcedefinition.apiextensions.k8s.io/vaultstaticsecrets.secrets.hashico
 
 Refer to the [VSO helm chart](/vault/docs/platform/k8s/vso/helm)
  overview for a full list of supported chart values.
+
+## Installation using kustomize
+
+You can install and update your installation using `kustomize` which allows you to extend the `config/` path of the VSO repository using kustomize primitives.
+
+To install using kustomize, download and untar/unzip the latest release from the [Releases Page](https://github.com/hashicorp/vault-secrets-operator/releases).
+```shell-session
+# Example
+$ wget -q https://github.com/hashicorp/vault-secrets-operator/archive/refs/tags/v0.4.3.tar.gz
+$ tar -zxf v0.4.3.tar.gz
+$ cd vault-secrets-operator-0.4.3/
+```
+
+Next install using `kustomize build`:
+```shell-session
+$ kustomize build config/default | kubectl apply -f -
+namespace/vault-secrets-operator-system created
+customresourcedefinition.apiextensions.k8s.io/hcpauths.secrets.hashicorp.com created
+customresourcedefinition.apiextensions.k8s.io/hcpvaultsecretsapps.secrets.hashicorp.com created
+customresourcedefinition.apiextensions.k8s.io/vaultauths.secrets.hashicorp.com created
+customresourcedefinition.apiextensions.k8s.io/vaultconnections.secrets.hashicorp.com created
+customresourcedefinition.apiextensions.k8s.io/vaultdynamicsecrets.secrets.hashicorp.com created
+customresourcedefinition.apiextensions.k8s.io/vaultpkisecrets.secrets.hashicorp.com created
+customresourcedefinition.apiextensions.k8s.io/vaultstaticsecrets.secrets.hashicorp.com created
+serviceaccount/vault-secrets-operator-controller-manager created
+role.rbac.authorization.k8s.io/vault-secrets-operator-leader-election-role created
+clusterrole.rbac.authorization.k8s.io/vault-secrets-operator-manager-role created
+clusterrole.rbac.authorization.k8s.io/vault-secrets-operator-metrics-reader created
+clusterrole.rbac.authorization.k8s.io/vault-secrets-operator-proxy-role created
+rolebinding.rbac.authorization.k8s.io/vault-secrets-operator-leader-election-rolebinding created
+clusterrolebinding.rbac.authorization.k8s.io/vault-secrets-operator-manager-rolebinding created
+clusterrolebinding.rbac.authorization.k8s.io/vault-secrets-operator-proxy-rolebinding created
+configmap/vault-secrets-operator-manager-config created
+service/vault-secrets-operator-controller-manager-metrics-service created
+deployment.apps/vault-secrets-operator-controller-manager created
+$ k get pods -n vault-secrets-operator-system
```

Comment:
May want a little whitespace here, or could even move the `get pods` command to a new box.

Also we should probably use `kubectl` instead of aliases like `k`.

```suggestion

$ kubectl get pods -n vault-secrets-operator-system
```